### PR TITLE
Khasan

### PR DESCRIPTION
**Pull request by khasanby**

### CODE REVIEW

File: DataLayer/Repository/IRepository/IRepository.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataLayer.Repository.IRepository
+{
+    internal interface IRepository
+    {
```
Comment (apilatau) [inline DataLayer/Repository/IRepository/IRepository.cs:10]: why do you push the empty files?

File: DataLayer/Repository/Repository.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataLayer.Repository
+{
+    internal class Repository : DataLayer.Repository.IRepository.IRepository
+    {
```
Comment (apilatau) [inline DataLayer/Repository/Repository.cs:10]: What is it needed for?

File: DataLayer/Repository/Repository.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataLayer.Repository
+{
+    internal class Repository : DataLayer.Repository.IRepository.IRepository
```
Comment (apilatau) [inline DataLayer/Repository/Repository.cs:9]: why internal?

File: TelegramBotAPI/Controllers/WebHookController.cs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+﻿using BookBot.Services;
+using Microsoft.AspNetCore.Mvc;
+using Telegram.Bot.Types;
+
+namespace TelegramBotAPI.Controllers
+{
+    public class WebhookController : ControllerBase
+    {
+        [HttpPost]
+        public async Task<IActionResult> Post([FromServices] HandleUpdateService handleUpdateService,
```
Comment (apilatau) [inline TelegramBotAPI/Controllers/WebHookController.cs:10]: What for?

**Review by apilatau — CHANGES_REQUESTED** [review]

**Comment by apilatau** [comment]
1) create a new branch **properly** according to your  current tasks
2) each task = the separate branch
2) resolve comments

File: DataLayer/Repository/IRepository/IRepository.cs (hunk 1 above)
Comment (khasanby) [reply DataLayer/Repository/IRepository/IRepository.cs:10]: Okay, I cloned the repository and added Coutry and City models and pushed it. I have not any branches yet. Did you expect me to write all code in here?

File: DataLayer/Repository/Repository.cs (hunk 2 above)
Comment (khasanby) [reply DataLayer/Repository/Repository.cs:10]: it is data layer, that is what I cloned from dev

**Pull request closed without merging**